datainterface/hikingtrails.js

```diff
@@ -4,14 +4,159 @@ const ObjectId = require('mongodb').ObjectId;
 require('dotenv').config()
 
 const uri =
-`mongodb+srv://${process.env.DB_USER}:${process.env.DB_PASSWORD}@cluster0.jymhgca.mongodb.net/?retryWrites=true&w=majority`
-
-
+`mongodb+srv://d-team:Test12@cluster0.f4ghe7b.mongodb.net/?retryWrites=true&w=majority`
 
 const client = new MongoClient(uri);
 
 const databaseName = 'hiking_db';
-const hikesCollName = 'Hikes';
+const trailsCollName = 'trails';
+const commCollName = 'comments';
+const parkingCollName = 'parking';
+
+module.exports = {}
+
+// HIKING TRAIL ENDPOINTS
+
+// #2 GET ALL HIKING TRAIL DATA
+module.exports.getAll = async () => {
+    const database = client.db(databaseName);
+    const trails = database.collection(trailsCollName);
+  
+    const query = {};
+    let trailsCursor = await trails.find(query);
```

Comment:
Is there any concern that there will be so many Trail records that this will become a slow, data intensive query? Should the number of results be limited?

Reply:
I added a limit of 100 trails